tests/src/Functional/AlertConfirmationTest.php

```diff
@@ -63,6 +63,7 @@ public function testAlertConfirmation() {
     $this->getSession()->getPage()->pressButton('Confirm');
     $this->assertSession()->pageTextContains('The alert banner ' . $title . ' has been set live');
 
+    $this->drupalGet($alert->toUrl('canonical')->toString());
```

Comment:
I'm fine with this. I kinda deliberately didn't put the url in to allow it to go to any alert/{id}/{action} as long as it has the feedback and the tab to remove.

Reply:
Removing it gives this error
>  ✘ Alert confirmation
   │
   │ The link Remove banner was not found on the page.
   │ Failed asserting that an array has the key 0.
   │ 
   │ /app/web/core/tests/Drupal/Tests/UiHelperTrait.php:496
   │ /app/web/modules/contrib/localgov_alert_banner/tests/src/Functional/AlertConfirmationTest.php:67
   │ 

Does that mean the test is broken or that the page they land on should have the remove link?
I think at the monent saving get redirected back to the collection form.